string command
------------------
string message
bool success
